app/build.gradle

```diff
@@ -74,8 +73,8 @@ android {
         production {
             applicationId "org.piramalswasthya.sakhi"
             // TMC Server Production API Base URL
-            buildConfigField "String", "BASE_TMC_URL", "\"https://assamtmc.piramalswasthya.org/\""
-
+         //   buildConfigField "String", "BASE_TMC_URL", "\"https://assamtmc.piramalswasthya.org/\""
+            buildConfigField "String", "BASE_TMC_URL", "\"https://amritdemo.piramalswasthya.org/\""
```

Comment:
@NikhilFegade is this the only way to change baseURL?
It looks like production base URL is changed.